cmd/finch/support_bundle.go

```diff
@@ -0,0 +1,99 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package main
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/runfinch/finch/pkg/command"
+	"github.com/runfinch/finch/pkg/flog"
+	"github.com/runfinch/finch/pkg/lima"
+	"github.com/runfinch/finch/pkg/support"
+)
+
+func newSupportBundleCommand(logger flog.Logger, builder support.BundleBuilder, lcc command.LimaCmdCreator) *cobra.Command {
+	supportBundleCommand := &cobra.Command{
+		Use:   "support-bundle",
+		Short: "Gathers logs and info into a bundle to help debug issues",
+	}
+	supportBundleCommand.AddCommand(
+		newSupportBundleGenerateCommand(logger, builder, lcc),
+	)
+	return supportBundleCommand
+}
+
+func newSupportBundleGenerateCommand(logger flog.Logger, builder support.BundleBuilder, lcc command.LimaCmdCreator) *cobra.Command {
+	supportBundleGenerateCommand := &cobra.Command{
+		Use:   "generate",
+		Args:  cobra.NoArgs,
+		Short: "Generate support bundle",
+		RunE:  newGenerateSupportBundleAction(logger, builder, lcc).runAdapter,
+	}
+
+	supportBundleGenerateCommand.Flags().StringArray("include", []string{}, "additional files to include in the support bundle")
```

Comment:
It is not clear what to put in --include and --exclude. Should it be absolute path or relative path?

Reply:
include can be either a relative or absolute path, exclude can be an absolute or relative path or the basename of a file. I'll add some more words here

Reply:
Will it cause lots of complicated scenarios if we support both "basename" or "relative path"? What if there is a "finch.yaml" in the current relative path? Will the command exclude the one in current relative path or ~/.finch/finch.yaml in that case?

I think we need to either try to remove the uncertain branches to simply the scenarios or cover them in e2e tests and docuemntation/CLI description.

Reply:
> Will the command exclude the one in current relative path or ~/.finch/finch.yaml in that case?

In that case if it's specified by basename, all files named `finch.yaml` will be excluded. If a user wants to exclude just one of the specific files, they can specify the full absolute or relative path to the file. I'll add some documentation on that to the CLI option.